packages/cli/cloud/src/login/action.ts

```diff
@@ -0,0 +1,135 @@
+import axios, { AxiosResponse, AxiosError } from 'axios';
+import { tokenServiceFactory, cloudApiFactory } from '../services';
+
+import type { CloudCliConfig, CLIContext } from '../types';
+
+const openModule = import('open');
```

Comment:
are you using the ESM only version ?

Reply:
Yes, this package only supports ESM. https://github.com/sindresorhus/open

Reply:
We already use open in the codebase and have locked the older version that has CJS btw.